src/xebia_email_signature/templates/new_form.html

```diff
@@ -0,0 +1,517 @@
+                        type="text"
+                        id="cta-0-link"
+                        name="cta[0][link]"
+                        value="https://www.xebia.com/news"
+                      />
+                    </div>
+                    <div id="cta-link-error" class="error"></div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="form-group">
+            <button
+              class="btn btn-md btn-dawn w-100 text-left font-semibold js-form-clone-btn"
+              data-clone-target=".js-form-cta"
+            >
+              <img src="static/icon-btn-add.svg" alt="Icon add" />
+              Add another Call to Action
+            </button>
+          </div>
+
+          <div class="form-bottom">
+            <a
+              class="btn btn-outline btn-small"
+              href="static/design.pdf"
+              target="_blank"
+            >
+              <img src="/static/icon-watch.svg" alt="Icon Watch" />
+              Design Guidelines
+            </a>
+
+            <input class="btn btn-primary" type="submit" value="Create" />
+          </div>
+        </form>
+      </div>
+      <div class="preview-container hidden">
+        <h2 class="preview-header">Preview:</h2>
+        <iframe
+          class="preview-iframe"
+          name="preview-iframe"
+          src="/new/signature"
+          height="100%"
+          width="100%"
+        ></iframe>
+      </div>
+    </main>
+
+    <script src="https://cdn.jsdelivr.net/npm/choices.js@10.2.0/public/assets/scripts/choices.min.js"></script>
+    <script>
+      function validateForm() {
+        let errors = document.getElementsByClassName('error');
+        for (let i = 0; i < errors.length; i++) {
+          errors[i].innerHTML = '';
+        }
+        let name = document.getElementById('full_name').value;
+        if (name.trim() === '') {
+          document.getElementById('full_name-error').innerHTML =
+            'Please enter your name';
+          return false;
+        }
+        let email = document.getElementById('email').value;
+        if (!validateEmail(email)) {
+          document.getElementById('email-error').innerHTML =
+            'Please enter a valid email address';
+          return false;
+        }
+
+        let job_role = document.getElementById('job_role').value;
+        if (job_role.trim() === '') {
+          document.getElementById('job_role-error').innerHTML =
+            'Please enter your role';
+          return false;
+        }
+
+        let phone = document.getElementById('phone').value;
+        if (!validatePhoneNumber(phone)) {
+          document.getElementById(
+            'phone-error'
+          ).innerHTML = `phone number (${phone} should start with a + followed by 10 to 15 digits without any other characters`;
+          return false;
+        }
+
+        return true;
+      }
+
+      function prefillEmail(full_name) {
```

Comment:
The pre-fill email no longer works in the new form

Reply:
It should work now(there is another PR, so i will be glad if you look into it), as we removed prefilled values used for testing and added hints.